README.md

```diff
@@ -1,16 +1,13 @@
-Shopify Universal Variable Templates
+Qubit Event Templates
 ====================================
-
-Universal Variable provides a standardised JSON structure for data relating to the user journey of your visitors, such user details, products viewed and transactions. Data contained in Universal Variable are pushed automatically to Qubit’s platform and Opentag scripts.
-
-The code in this repository will take away 95% of the development work required in implementing Universal Variable on your Shopify store.
+The code in this repository will take away 95% of the development work required in implementing Qubit on your Shopify store.
 
 #Overview
 
-When implementing [Universal Variable](http://tools.qubitproducts.com/uv) and [Opentag](http://www.qubitproducts.com/tag-management), you need to do two things from a technical perspective:
+When implementing Qubit, you need to do two things from a technical perspective:
 
-1. Add the Opentag container to the `<head>` of every page
-2. Add the Universal Variable (UV) directly before the Opentag container on every page.
+1. Add the Smartserve container to the `<head>` of every page
+2. Emit emits based on user browsing behaviour
```

Comment:
Does the UV API need to be added in the head, or are we assuming that Smartserve will load before the events start firing?